pkg/utils/managedresources/builder/secrets.go

```diff
@@ -87,6 +87,17 @@ func (s *Secret) WithAnnotations(annotations map[string]string) *Secret {
 	return s
 }
 
+// AddAnnotations adds the specified annotations to the secret's existing annotations.
```

Comment:
```suggestion
// AddAnnotations adds the annotations to the existing secret annotations.
```

Nit, to be consistent with `AddLabels`.